en/app.ftl

```diff
@@ -250,12 +257,16 @@ faq-question-availability-answer = Free { -brand-name-relay } is available in mo
 
 faq-question-4-question = Can I reply to messages using my { -brand-name-relay } alias?
 faq-question-4-answer = { -brand-name-relay-premium } users can reply to a forwarded email within 3 months of receiving the email. Any CC’ed or BCC’ed addresses will not be included in your replies.
+# Deprecated
 faq-question-4-answer-v2 = { -brand-name-relay-premium } users can reply to a forwarded email within 3 months of receiving the email. If you add a CC or BCC when you reply back to an email, your original email address will be exposed to the receivers and those on the email. If you do not want your original email address exposed, do not add CCs or BCCs when replying.
+faq-question-4-answer-v3 = { -brand-name-relay-premium } users can reply to a forwarded email within 3 months of receiving the email. If you add a CC or BCC when you reply back to an email, your original email address will be exposed to the recipient and those on the email. If you do not want your original email address exposed, do not add CCs or BCCs when replying.
 faq-question-4-answer-html = { -brand-name-relay } does not yet offer the ability to reply using an alias.
     If you try, nothing will happen. We are planning an additional feature to let you <a href="{ $url }" { $attrs }>reply anonymously to senders</a>.
 
 faq-question-subdomain-characters-question = What characters can I use to create a subdomain?
+# Deprecated
 faq-question-subdomain-characters-answer = Only lower case English alphabet, numbers, and dashes (-).
+faq-question-subdomain-characters-answer-v2 = You can only use lower case English alphabet, numbers, and hyphens to create a subdomain.
```

Comment:
Shall we use Latin alphabet instead? When we say English alphabet, we mean 26 letters but nothing outside of the list?

Reply:
Hmm, I think so, but let me verify with content (also for your other remark).

Reply:
New language is: 
> You can only use lower-case English letters, numbers, and hyphens to create a subdomain.